src/components/AddSongMenu.tsx

```diff
@@ -0,0 +1,226 @@
+import Box from "@mui/material/Box";
+import Drawer from "@mui/material/Drawer";
+import Button from "@mui/material/Button";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+import {
+  Avatar,
+  CircularProgress,
+  InputBase,
+  ListItemAvatar,
+  Paper,
+  Typography,
+} from "@mui/material";
```

Comment:
Is there a reason the import styles are different? Would expect only either the destructuring approach or the sub directly approach for consistency.

Reply:
I was just not very mindful of the way I was importing, thanks.